src/rdbfdx.f

```diff
@@ -26,6 +26,8 @@
 C> @author Woollen @date 1994-01-06
       SUBROUTINE RDBFDX(LUNIT,LUN)
 
+        use bufrlib
```

Comment:
Can you explain why we need the use statement now? We didn't need it before...

Reply:
We're trying to modernize the C-Fortran interface and get rid of the UNDERSCORE and other legacy stuff, so we now have wrappers with `BIND(C,` defined in a module interface in bufrlib.F90.  So that's why we need to explicitly include the new module whenever we call these C routines from Fortran inside of the library.

This is similar to how all of the test codes now need to `use bufr_interface` when calling C from Fortran, or `#include bufr_interface.h` when calling Fortran from C.

Reply:
OK, that makes sense. SOrry, I'm a little slow sometimes! ;-)